Sources/TuistSigning/SigningInstaller.swift

```diff
@@ -1,37 +1,29 @@
 import Foundation
 import TSCBasic
+import TuistCore
 import TuistSupport
 
-enum SigningInstallerError: FatalError, Equatable {
-    case noFileExtension(AbsolutePath)
-    case provisioningProfilePathNotFound(ProvisioningProfile)
-    case revokedCertificate(Certificate)
-    case expiredProvisioningProfile(ProvisioningProfile)
-
-    var type: ErrorType {
-        switch self {
-        case .noFileExtension, .provisioningProfilePathNotFound, .revokedCertificate, .expiredProvisioningProfile:
-            return .abort
-        }
+extension LintingIssue {
+    static func noFileExtension(_ path: AbsolutePath) -> Self {
+        Self(reason: "Unable to parse extension from file at \(path.pathString)", severity: .error)
     }
 
-    var description: String {
-        switch self {
-        case let .noFileExtension(path):
-            return "Unable to parse extension from file at \(path.pathString)"
-        case let .revokedCertificate(certificate):
-            return "The certificate has been revoked \(certificate.name)"
-        case let .expiredProvisioningProfile(provisioningProfile):
-            return "The provisioning profile \(provisioningProfile.name) has expired"
-        case let .provisioningProfilePathNotFound(provisioningProfile):
-            return "Could not find any path for \(provisioningProfile.name)"
-        }
+    static func expiredProvisioningProfile(_ profile: ProvisioningProfile) -> Self {
+        Self(reason: "The provisioning profile \(profile.name) has expired", severity: .warning)
     }
+
+//    static func revokedCertificate(_ certificate: Certificate) -> Self {
+//        Self(reason: "The certificate has been revoked \(certificate.name)", severity: .warning)
+//    }
+//
+//    static func provisioningProfilePathNotFound(_ profile: ProvisioningProfile) -> Self {
+//        Self(reason: "Could not find any path for \(profile.name)", severity: .warning)
+//    }
```

Comment:
this was unused. not sure if it can be deleted.

Reply:
I'd go ahead then and delete them